### PR TITLE
chore(profiling): Add end of profiling beta alert

### PR DESCRIPTION
**Pull request by Zylphrex**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Clarify what happens to the profiles sent after the beta closed.

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by indragiek — APPROVED** [review]

File: src/docs/product/profiling/getting-started.mdx [hunk 1]
```diff
@@ -7,6 +7,12 @@ redirect_from:
 description: 'Get started with Profiling, which allows you to see code-level profiling information for your Sentry apps.'
 ---
 
+<Note>
+
+The beta program for Profiling is closed. Profiles sent during this period will not appear in your dashboard if you did not participate in the beta program. Profiling will be generally available soon.
+
```
Comment (lizokm) [inline src/docs/product/profiling/getting-started.mdx:13]:
```suggestion

The beta program for Profiling is now closed, but Profiling will become generally available soon. If you weren't part of the beta program, any Profiles sent during this time won't appear in your dashboard.

```

File: src/platforms/common/profiling/index.mdx [hunk 2]
```diff
@@ -29,6 +29,12 @@ notSupported:
 description: 'Learn how to enable profiling in your app if it is not already set up.'
 ---
 
+<Note>
+
+The beta program for Profiling is closed. Profiles sent during this period will not appear in your dashboard if you did not participate in the beta program. Profiling will be generally available soon.
+
+</Note>
```
Comment (lizokm) [inline src/platforms/common/profiling/index.mdx:36]:
```suggestion
<Note>

The beta program for Profiling is now closed, but Profiling will become generally available soon. If you weren't part of the beta program, any Profiles sent during this time won't appear in your dashboard.

</Note>
```

File: src/platforms/node/common/profiling/index.mdx [hunk 3]
```diff
@@ -3,6 +3,12 @@ title: Profiling
 description: 'Learn more about how to configure our Profiling integration and start profiling your code.'
 ---
 
+<Note>
+
+The beta program for Profiling is closed. Profiles sent during this period will not appear in your dashboard if you did not participate in the beta program. Profiling will be generally available soon.
+
```
Comment (lizokm) [inline src/platforms/node/common/profiling/index.mdx:9]:
```suggestion

The beta program for Profiling is now closed, but Profiling will become generally available soon. If you weren't part of the beta program, any Profiles sent during this time won't appear in your dashboard.

```

File: src/platforms/php/common/profiling/index.mdx [hunk 4]
```diff
@@ -5,6 +5,12 @@ description: 'Learn more about how to configure our Profiling integration and st
 
 <Note>
 
+The beta program for Profiling is closed. Profiles sent during this period will not appear in your dashboard if you did not participate in the beta program. Profiling will be generally available soon.
+
```
Comment (lizokm) [inline src/platforms/php/common/profiling/index.mdx:9]:
```suggestion
The beta program for Profiling is now closed, but Profiling will become generally available soon. If you weren't part of the beta program, any Profiles sent during this time won't appear in your dashboard.

```

File: src/platforms/php/guides/laravel/profiling/index.mdx [hunk 5]
```diff
@@ -5,6 +5,12 @@ description: 'Learn more about how to configure our Profiling integration and st
 
 <Note>
 
+The beta program for Profiling is closed. Profiles sent during this period will not appear in your dashboard if you did not participate in the beta program. Profiling will be generally available soon.
+
```
Comment (lizokm) [inline src/platforms/php/guides/laravel/profiling/index.mdx:9]:
```suggestion
The beta program for Profiling is now closed, but Profiling will become generally available soon. If you weren't part of the beta program, any Profiles sent during this time won't appear in your dashboard.

```

File: src/platforms/php/guides/symfony/profiling/index.mdx [hunk 6]
```diff
@@ -5,6 +5,12 @@ description: 'Learn more about how to configure our Profiling integration and st
 
 <Note>
 
+The beta program for Profiling is closed. Profiles sent during this period will not appear in your dashboard if you did not participate in the beta program. Profiling will be generally available soon.
+
```
Comment (lizokm) [inline src/platforms/php/guides/symfony/profiling/index.mdx:9]:
```suggestion
The beta program for Profiling is now closed, but Profiling will become generally available soon. If you weren't part of the beta program, any Profiles sent during this time won't appear in your dashboard.

```

**Review by lizokm — APPROVED** [review]
Made a small tweak, but otherwise LGTM!

**Pull request merged**